src/custom_nodes/model_zoo_intel_object_detection/model_zoo_intel_object_detection.cpp

```diff
@@ -525,6 +527,15 @@ int getOutputsInfo(struct CustomNodeTensorInfo** info, int* infoCount, const str
     (*info)[2].dims[1] = 1;
     (*info)[2].dims[2] = 1;
     (*info)[2].precision = FP32;
+
```

Comment:
This is not resolved correctly. There is separate task to do it correctly: https://jira.devtools.intel.com/browse/CVS-69864
If we merge this code we will have memory leak or double free issue.

On main, the model zoo custom node does not use internal manager, therefore it does not use preallocated buffers. On develop, it uses preallocated buffers but you do not use it for info[3]. What's more, there will be segfault due to writing to unallocated memory (setting `*outputsCount = 3;` but writting to 4th element).